book/06-github/sections/2-contributing.asc

```diff
@@ -71,7 +71,8 @@ $ cd blink
 $ git checkout -b slow-blink <2>
 Switched to a new branch 'slow-blink'
 
-$ sed -i '' 's/1000/3000/' blink.ino <3>
+$ sed -i '' 's/1000/3000/' blink.ino <3> (MacOSX)
+$ sed -i 's/1000/3000/' blink.ino <3> (Linux)
```

Comment:
Let's format this a bit differently:

```shell
$ sed -i '' 's/1000/3000/' blink.ino <3>
# If you're on a Linux system, do this instead:
# $ sed -i 's/1000/3000/' blink.ino
```

That way the "3" bullet only shows up once.